app/src/main/java/org/flyve/inventory/agent/utils/HttpInventory.java

```diff
@@ -104,7 +104,7 @@ public void sendInventory(final String lastXMLResult, final OnTaskCompleted call
         }
 
         try {
-            url = new URL(mFusionApp.getUrl());
+            url = new URL("https://dev.flyve.org/glpi/plugins/fusioninventory/");
```

Comment:
Why it's better?

Reply:
At this point on instrumented test this var mFusionApp.getUrl() is not loaded with data